{"name": "Если авторизоваться под несуществующим пользователем, запрос возвращает ошибку", "status": "passed", "description": "Этот логин уже используется", "start": 1724774901670, "stop": 1724774901785, "uuid": "c9ebb663-9cbe-4eff-b4ca-5f1b077b3743", "historyId": "462b672807218c6ce490b65136a58276", "testCaseId": "462b672807218c6ce490b65136a58276", "fullName": "tests.test_login_courier.TestLoginCourier#test_authorization_defunct_courier", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login_courier"}, {"name": "subSuite", "value": "TestLoginCourier"}, {"name": "host", "value": "DESKTOP-7UGKSDC"}, {"name": "thread", "value": "18756-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login_courier"}]}